Animations/src/core/ImGuiLayer.cpp

```diff
@@ -34,13 +34,17 @@ namespace MathAnim
 			ImGuiIO& io = ImGui::GetIO();
 			io.ConfigFlags |= ImGuiConfigFlags_NavEnableKeyboard;       // Enable Keyboard Controls
 			io.ConfigFlags |= ImGuiConfigFlags_DockingEnable;           // Enable Docking
-			io.ConfigFlags |= ImGuiConfigFlags_ViewportsEnable;         // Enable Multi-Viewport / Platform Windows
+			// io.ConfigFlags |= ImGuiConfigFlags_ViewportsEnable;      // Enable Multi-Viewport / Platform Windows
```

Comment:
Once again, I'd rather leave this in as it's a feature I want to support. I'll be adding some editor preferences in the future to enable static layouts, but for now I don't want to remove this :)